src/main/java/frc/robot/commands/PlaceCoralAndTakeAlgae.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.*;
+
+public class PlaceCoralAndTakeAlgae extends SequentialCommandGroup {
+
+    private static final double speed = -0.2;
```

Comment:
UPPER_SNAKE_CASE
also,"speed" doesnt tell me much when you have 6 different subsystems